jetty-ee8/jetty-ee8-webapp/src/main/config/etc/jetty-ee8-deploy.xml

```diff
@@ -7,7 +7,7 @@
   <Ref refid="deploymentScanner">
     <Call name="configureEnvironment">
       <Arg>ee8</Arg>
-      <Set name="defaultContextHandlerClass">
+      <Set name="defaultContextHandlerClassName">
         <Property name="contextHandlerClass" default="org.eclipse.jetty.ee8.webapp.WebAppContext" />
```

Comment:
The property should be renamed to `defaultContextHandlerClass`, like it is in `static-deploy`.

Also, it should be added under [ini-template] in `ee8.mod`.